hours.yaml

```diff
@@ -0,0 +1,19 @@
+regular:
+  - ['9:00', '18:00']
+  - ['9:00', '20:00']
+  - ['9:00', '20:00']
+  - ['9:00', '20:00']
+  - ['9:00', '18:00']
+  - ['9:00', '20:00']
+  - ['9:00', '20:00']
```

Comment:
I'll add a comment that says this starts on Monday.

Reply:
Would it be possible to make it like
```
regular:
  -monday:
    - ['9:00', '18:00']
```
? I feel that it would be easier to make mistakes about which day of the week needs to be changed if you have to count from Monday every time.

Reply:
I thought about this, but I think it makes the loading code more complicated. I wouldn't be opposed to adding a comment on each day.

Reply:
this is the WIP format I was using in my hours re-write: https://github.com/abizer/ocflib/blob/8df85f517473c40d05cc74185ddf9fb16afed02b/ocflib/lab/hours.yaml

Reply:
A comment on each day sounds good too.